eventhandler.go

```diff
@@ -16,33 +16,67 @@ type EventHandler interface {
 	Topic() string
 }
 
-// SetupEventHandler sets up an Eventhandler to the event bus.
```

Comment:
ist das nicht ein breaking change?

Reply:
joa jain fast irgentwie, deswegen steht ja oben use SetupEventHandlers instead... ist der exakt gleiche "synthax" nur halt ein s im method namen mehr...

Reply:
ja schon, dennoch bricht es
also fände komisch, wenn wir in unseren eigenen libs breaking changes einbauen. es sei denn du machst daraus ne v3, aber das war hier wohl weniger der plan

Reply:
es wird auch bereits in code von uns genutzt

<img width="945" alt="image" src="https://github.com/user-attachments/assets/f5fd179a-66b1-471b-a03e-37f765caf859">

Reply:
wird es denn irgendwo genutzt? 
wenn noch komplett unused, von mir aus, aber da sollten wir sicher sein

Reply:
ja genau das wollte ich eig umgehen

Reply:
dann deprecated ich das und es kann beim nächsten major einfach raus